flax/linen/linear.py

```diff
@@ -481,6 +481,17 @@ class ConvLocal(_Conv):
   def shared_weights(self) -> bool:
     return False
 
+# get docstring from _Conv, delete leading spaces
```

Comment:
This works for RTD but I think static analysis tools won't understand this kind of docstring manipulation. I would prefer to keep things simple and just copy paste the attributes

Reply:
Got it, will do!